### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by ghiscoding** [ghiscoding/aurelia-nest-auth-mongodb#47] [Ideas]
@ductran2702 I came across your fork of my package, I see lot of interesting features in your fork (like confirmEmail, resetPassword, ...). I would be really happy to get a Pull Request with all of that, so feel free to contribute if you wish. I'm sure lot of other users would be interested as well. Also note that I just updated all npm packages to latest on both client & backend side.

 Cheers and Happy Holidays

**Comment by ductran2702** [comment]
@ghiscoding Glad to hear that from you. I will try to make a Pull Request when I have spare time (I'm a little bit busy now). Because there are some code that are specific in my project and not necessary need for other users.

Happy New Year.

**Reply by ghiscoding** [reply]
Awesome, feel free to do whenever the time is best for you. 

Thanks and Happy New Year as well 😃
